network/gateway.go

```diff
@@ -54,22 +54,21 @@ func getDefaultRouteLinux() (net.IP, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
+	logger.Tracef("ip route show output %q", output)
```

Comment:
Given the multi-line nature, this is probably better as:
logger.Tracef("ip route show output:\n%s", output)
Otherwise you end up with lots of "\n" escapes that are going to be hard to understand.